include/configuration.h

```diff
@@ -0,0 +1,18 @@
+#ifndef TCP_SERVER_CONFIGURATION_H
+#define TCP_SERVER_CONFIGURATION_H
+
+ typedef struct {
+    int PORT;
+    char PATH[512];
```

Comment:
The header `/usr/include/linux/limits.h` specifies a macro `PATH_MAX` for the maximum characters in a path.
On Ubuntu this is 4096.
(So the buffer needs to be PATH_MAX + 1 long.)

See [this blog](https://insanecoding.blogspot.com/2007/11/pathmax-simply-isnt.html) post on flaws of `PATH_MAX`. But I think 4096 is still more solid than any other arbitrary number.

Maybe an other name would be also more expressive? loadConfig also has the path of the configuration file. It could get a little confusing which path is which.